README.md

```diff
@@ -1,29 +1,41 @@
 # Azure Key Vault Secrets for Insomnia
+
+![Azure Key Vault Secrets for Insomnia Logo](https://unpkg.com/insomnia-plugin-azure-keyvault-secrets/plugin-background.png)
```

Comment:
mhmmm, the npm package has already got a cover in the [package.json](https://github.com/glconti/insomnia-plugin-azure-keyvault-secrets/blob/9e27ffc0dedb817a8c204e818c81d4be9b80a8d0/package.json#L14) which works on the Insomnia plugin page, but for example it's not visible on the [npm page ](https://www.npmjs.com/package/insomnia-plugin-azure-keyvault-secrets)

I think that if we add this here, we should remove the `cover` property from the `package.json` otherwise the image will be disploayed twice.

Reply:
I'll do it after this one is merged then :), thaks for the PR!